content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst

```diff
@@ -2,35 +2,74 @@
 Invoice Time Spent on Tickets (Postpaid Support Services)
 =========================================================
 
-Directly pull the billable time you have tracked on your helpdesk tickets into sales orders and
-invoices through a project task. It gives you more control over what you charge your client, and it
-is more efficient.
+Connect a Helpdesk ticket to a Project task. While working on the ticket, use either the ticket or 
+the task to record timesheets. When the ticket is solved, Odoo can pull from those timesheets to 
+bill the client. This integration provides more control and transparency over what the client is 
+charged for.
 
 Configuration
 =============
 
-| Go to :menuselection:`Helpdesk --> Configuration --> Helpdesk team --> Edit` and enable the options
-  *Timesheet on Ticket* and *Time Reinvoicing*.
-| Under *Timesheet on Ticket*, choose the *Project* to which tickets (and timesheets) will be
-  linked by default. Open its *External link* to enable the feature *Bill from tasks*.
+Go to :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams`. Select a Helpdesk team and 
+click :guilabel:`Edit`. Then, enable the options :guilabel:`Timesheet on Ticket` and 
+:guilabel:`Time Reinvoicing`.
 
-.. image:: media/reinvoice_time1.png
+In the :guilabel:`Timesheet on Ticket` drop-down option, choose the Project to which tickets (and 
+timesheets) will be linked by default. Open the Project's external link to edit its settings and 
+enable the :guilabel:`Billable` option. Save both the Project and the Helpdesk team's settings to 
+apply these features.
+
+.. image:: reinvoice_time/project_billable.png
+   :align: center
+   :alt: Billable feature in Project settings
+
+Create a sales order
+====================
+
+After a ticket comes into the pipeline, go to :menuselection:`Sales --> Create`. Add the customer
+from the ticket to the new quotation. In the :guilabel:`Order Lines` tab, add the 
+:guilabel:`Service on Timesheet` product. Click :guilabel:`Confirm` to turn the quotation into a 
+sales order. 
+
+.. tip::
+   Remember the :abbr:`SO(Sales Order)` name for later to link the Helpdesk ticket to the correct 
+   sales order.
+
+.. image:: reinvoice_time/service_on_timesheet.png
    :align: center
-   :alt: Bill from Tasks in Odoo Helpdesk
+   :alt: Add Service on Timesheet product to the SO
 
-Create a sales order and an invoice
-====================================
+Record a timesheet
+==================
 
-Now, once you have recorded the time you spent on the helpdesk ticket, under the *Timesheets* tab,
-access the task clicking on its name.
+Jump back to the Helpdesk ticket by going to the Helpdesk dashboard and clicking 
+:guilabel:`Tickets` on the team's card. Another way to access the ticket is by going to the 
+Projects dashboard and clicking :guilabel:`Tickets` on the project card. Click :guilabel:`Edit` 
```

Comment:
Think we can do something with this sentence that starts with "Another way..."

I dont know if it warrants a `:tip:` directive because that would break up the flow of the instruction.

Maybe something like this?

(Also, I would consider using more time order flow words to help communicate linear and conditional actions...)

```suggestion
:guilabel:`Tickets` on the team's card (or by going to the 
Projects dashboard and clicking :guilabel:`Tickets` on the project card). Then click :guilabel:`Edit` 
```